### DISCUSSION TITLE
MariaDB Temporal Data Tables Support for Prisma Introspect

### DISCUSSION THREAD

**Discussion by saraveti** [prisma/prisma#3333] [Q&A] [answered]
I really liked many features in Prisma and I want use it in my next project. I want to use the Prisma with Maria DB. 

I have used `npx primsa introspect` which is not recognizing the 'SYSTEM VERSIONED' tables. This feature is called 'Temporal Data Tables'. Here is the [link](https://mariadb.com/kb/en/temporal-data-tables/) to Maria DB documentation. 

This is what I have specified in prisma.schema for data source.

datasource db {
  provider = "mysql"
  url      = env("DATABASE_URL")
}

am I missing any thing here. Or is this part of your future enhancement item. 

Please do respond this is very critical item to decide whether to choose the Prisma or not.

**Comment by ryands17** [comment, marked as answer]
Hey @saraveti 👋 
It would be great if you could create a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) so that we can look into this.

**Reply by saraveti** [reply]
Thanks for the response @ryands17 . I have created a new feature request. #3334

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
